docs/develop/terra-js/common-examples.md

```diff
@@ -31,47 +33,48 @@ const lcd = new LCDClient({
 
 ```js
 // Replace with address to check.
-const address = 'terra1xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx';
+const address = "terra1xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx";
 const [balance] = await lcd.bank.balance(address);
 console.log(balance.toData());
 ```
 
-Example response: 
+Example response:
 
 ```js
-[
-  { denom: 'uluna', amount: '5030884' },
-  { denom: 'uusd', amount: '433108784' }
-]
+[{ denom: "uluna", amount: "5030884" }];
 ```
 
 ## Get wallet balance (CW20 tokens)
 
 ```js
-// ANC on bombay-12
-const tokenAddress = 'terra1747mad58h0w4y589y3sk84r5efqdev9q4r02pc';
-const walletAddress = 'terra1f44ddca9awepv2rnudztguq5rmrran2m20zzd6';
-const response = await lcd.wasm.contractQuery(tokenAddress, { balance: { address: walletAddress }});
+// ANC on pisco-1
```

Comment:
There won't be ANC on Terra 2.0

Should we change this, or just leave as an example?
I see merit either way.